cllc-public-app/ClientApp/src/app/models/licensee-change-log.model.ts

```diff
@@ -512,155 +513,7 @@ export class LicenseeChangeLog {
     return changed;
   }
 
-  public static FixLicenseeChangeLogArray(data: LicenseeChangeLog[]): LicenseeChangeLog[] {
```

Comment:
I'm concerned about removing this and the next block of code.  Have you fully tested it with this not in place?  We had to put this in due to problems.